packages/ibm-products/src/components/EmptyStates/_empty-state.scss

```diff
@@ -73,7 +74,7 @@ $block-class: #{c4p-settings.$pkg-prefix}--empty-state;
 }
 
 .#{$block-class} .#{$block-class}__action-button {
-  display: block;
+  display: inline-block;
```

Comment:
I think the `inline-block` should be on instead of the action button, otherwise we still see the overly large focus area.

<img width="295" alt="image" src="https://github.com/carbon-design-system/ibm-products/assets/1717398/053987ad-2f52-483d-8713-307bd49d26f4">


```scss
.#{$block-class}--empty-state .#{$block-class}--empty-state__link {
}
```